torch/csrc/jit/mobile/flatbuffer_loader.h

```diff
@@ -61,6 +61,12 @@ class FlatbufferLoader {
       IValueParser parser);
   mobile::Module parseModule(mobile::serialization::Module* module);
 
+  typedef TypePtr (*TypeResolver)(
+      const std::string& type_str,
+      std::shared_ptr<CompilationUnit> cu);
+
+  void internal_registerTypeResolver(TypeResolver type_resolver);
```

Comment:
If this is only meant for internal, could we consider making it private or protected?

Reply:
it need to be called outside of FlatbufferLoader class.